README.md

```diff
@@ -12,16 +12,13 @@
 <tr><th>
 Browsers
 </th><td width=100%>
-
 Load `@octokit/plugin-rest-endpoint-methods` and [`@octokit/core`](https://github.com/octokit/core.js) (or core-compatible module) directly from [cdn.pika.dev](https://cdn.pika.dev)
-
 ```html
 <script type="module">
   import { Octokit } from "https://cdn.pika.dev/@octokit/core";
   import { restEndpointMethods } from "https://cdn.pika.dev/@octokit/plugin-rest-endpoint-methods";
 </script>
 ```
-
```

Comment:
here you removed 3 existing new lines. You did not edit your changes at https://github.com/octokit/plugin-rest-endpoint-methods.js/pull/265 but you edited the original `README.md` file (before your changes from #265).

I'll close this PR, I've left instructions on how to edit your existing PR, let's discuss there, ok?